### PR TITLE
Background Segmentation using Pretrained MaskRCNN with Resizing

### PR DESCRIPTION
**Pull request by markus-hinsche**
# Description

**Background segmentation** using pre-trained MaskRCNN, then using the mask, applied binary segmentation to mask out the child from the background. The final result is in image format with child as `1` and background as `0` which I am display on the system.

User story: https://dev.azure.com/cgmorg/ChildGrowthMonitor/_workitems/edit/301

# Results

Damnik(Omdena) has tested this out on the anon_rgb_training dataset on Azure on different poses. Poses like `100` & `200` were giving good results, where the child is aligned straight to the camera. In some of the cases, the mask was not able to cover the child fully, some portion was left out.

# Checklist:

- [x] I have tested the code on Azure under Users/jaindamnik203 folder
- [x] The code is formatted & aligned.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests
- [x] New and existing unit tests pass locally with my changes
- [x] My Code is review by 1 People

### CODE REVIEW

**Comment by markus-hinsche** [comment]
Adjusted https://github.com/Welthungerhilfe/cgm-ml/pull/165
